src/lib.rs

```diff
@@ -1,11 +1,11 @@
 //! # halo2
 
 #![cfg_attr(docsrs, feature(doc_cfg))]
-// Build without warnings on stable 1.51 and later.
+// Build without warnings on stable 1.54 and later.
 #![allow(unknown_lints)]
-// Disable old lint warnings until our MSRV is at least 1.51.
+// Disable old lint warnings until our MSRV is at least 1.54.
```

Comment:
I think this is not a correct comment change. We should revisit whether this is necessary on Rust 1.54+.